cli/README.md

```diff
@@ -16,43 +19,59 @@ To use Light CLI, you need to have Node.js (version 12 or later) and npm (Node P
    ```shell
    light --version
    ```
+   
+## Usage
+
+The CLI lets you initialize a PSP scaffold project and customize, build, and test it. (```light psp:...``` subcommand)
+
+It also provides utilities for local development, such as running a local Solana-test-validator with all necessary Light accounts and pre-loaded programs. (```light test-validator```).
+
+You can also execute common user actions such as shielding and sending private transfers. (```light shield```, ```light transfer```, ```light unshield```).
+
+For the full list of available commands, see below:
+
 
-## Commands
+### Commands
 
 - `help`: Display help information.
-- `init`: Initialize a new PSP project.
-- `build`: Build the PSP project.
-- `test`: Run the PSP Project tests.
 - `account`: Get the current account details
 - `airdrop`: Perform a native Solana or SPL airdrop to a user.
 - `balance`: Retrieve the balance, inbox balance, or utxos for the user.
 - `config`: Update the configuration values.
 - `history`: Retrieve transaction history for the user.
 - `accept-utxos`: Merge multiple utxos into a single UTXO.
-- `test-validator`: Perform setup tasks.
 - `shield`: Shield tokens for a user.
   - `shield:sol`: Shield sol for a user.
   - `shield:spl`: Shield spl tokens for a user.
 - `transfer`: Transfer tokens to a recipient.
 - `unshield`: Unshield tokens for a user.
   - `unshield:sol`: Unshield sol for a user.
   - `unshield:spl`: Unshield spl tokens for a user.
+
+- `test-validator`: Starts a solana-test-validator and performs all setup tasks. Use in a separate terminal instead of solana-test-validator.
```

Comment:
``
```suggestion
- `test-validator`: Starts a solana-test-validator and with an initialized light environment. Use in a separate terminal instead of solana-test-validator.
```